package-lock.json

```diff
@@ -3318,7 +3318,6 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
-        "fsevents": "^1.2.7",
```

Comment:
Remove this diff?

Reply:
I don't know why that is even in there 🤔